clients/python/girder_client.py

```diff
@@ -434,7 +437,7 @@ def _transformFilename(self, name):
         name = name.replace(os.path.sep, ' ')
         if os.path.altsep:
             name = name.replace(os.path.altsep, ' ')
-        return re.sub(r'^[/\\]+', '', name)
+        return _safeNameRegex.sub('', name)
```

Comment:
That's a good point, I'll go ahead and make it the underscore, since I'm not sure about having a filename that is entirely whitespace. :)